### PR TITLE
First draft of CLI configuration documentation.

### PR DESCRIPTION
**Pull request by sptramer**

### CODE REVIEW

**Comment by formulahendry** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) | [Details](#user-content-latest%2Fdocs-ref-conceptual%2Fazure-cli-configuration.md)
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/advisor?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

<a name="latest%2Fdocs-ref-conceptual%2Fazure-cli-configuration.md"></a>
### [latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md)
 - **Line 72**: **[Warning]** Invalid file link:(~/latest/docs-ref-conceptual/aka.ms/AzureCliLegal).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C12%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712121704403329-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=dxuzDpI%2FlhH0wo%2FnkwjlzRZZaZOwgR8%2BWIYysz2w3dk%3D&st=2017-12-12T17%3A02%3A06Z&se=2018-01-12T17%3A07%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/advisor?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C13%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712131845380576-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=IonKCximOu3qo98zHvp%2BAHMuYV9V5G2x2fYfZcGVHRc%3D&st=2017-12-13T18%3A43%3A16Z&se=2018-01-13T18%3A48%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/advisor?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C13%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712131929565589-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=MM11L%2B07Ojw1fTcF44eEjMjDZU5uV%2F6IJH7hkURqbs8%3D&st=2017-12-13T19%3A27%3A41Z&se=2018-01-13T19%3A32%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/advisor?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C13%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712132111355826-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=lpnt%2BxoU9QQ%2BZIh3Q3mHP5vWF8xNee9GP1upoYZpvkE%3D&st=2017-12-13T21%3A09%3A07Z&se=2018-01-13T21%3A14%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: latest/docs-ref-conceptual/azure-cli-configuration.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+3. Values in the configuration file
+
+The configuration file itself is located at `$AZURE_CONFIG_DIR/config`. The default value of `AZURE_CONFIG_DIR` is `$HOME/.azure/config` on Linux and macOS,
+and `%USERPROFILE%\.azure\config` on Windows.
+
+The following table contains all of the sections and option names that can be placed in a configuration file. Their corresponding
+environment variables can be set as `AZURE_{section}_{name}`, as all caps. For example, the `location` value would be stored in the
+environment variable `AZURE_DEFAULT_LOCATION`.
+
+Any value which has a default available does not have to be present in the command line arguments, even if it is required. For example,
```
Comment (TomArcher) [inline latest/docs-ref-conceptual/azure-cli-configuration.md]: Should be: Any value that ...

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/advisor?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C14%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712141947038915-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=LM6eTHgKCvClnaNNehs%2BzX%2Fwc%2Fs5MoghYQOUS%2BE63J4%3D&st=2017-12-14T19%3A44%3A52Z&se=2018-01-14T19%3A49%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/network/application-gateway/probe?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C14%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712142350541427-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=y3gjKZGyuoKXHtC9dR5MUWQwnNe8H1nFFYC6kylZJvQ%3D&st=2017-12-14T23%3A48%3A16Z&se=2018-01-14T23%3A53%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/advisor?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C14%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712142353500257-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=z3g6zmdHI5eHVMadqyBoVDALZBywqbKDHBKZXLoChi4%3D&st=2017-12-14T23%3A51%3A07Z&se=2018-01-14T23%3A56%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/advisor?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C15%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712150028235861-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=KJGXnaXCVZ7fKxxtjefLaeQKQ5RHRshHc9uJ3S2eXc0%3D&st=2017-12-15T00%3A25%3A54Z&se=2018-01-15T00%3A30%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/advisor?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C15%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712150103037087-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=JpSgBP7YIBqydQteUu%2FX1aYlop9B1eRmuBqgM6ndUbs%3D&st=2017-12-15T01%3A00%3A28Z&se=2018-01-15T01%3A05%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/network/application-gateway/probe?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C15%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712150138321433-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=FGpsFKUzKWOqwVt4uZxe1%2FImSRQCcMhRramK7a3rtAE%3D&st=2017-12-15T01%3A35%3A55Z&se=2018-01-15T01%3A40%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/advisor?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C15%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712150203041175-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ox%2Bw4Mm%2BmVzUR4VcFbo6UukHjbCJo%2B5ABlx65SjSfJY%3D&st=2017-12-15T02%3A00%3A26Z&se=2018-01-15T02%3A05%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/advisor?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C15%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712150243377520-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zSrM56Lgiz3H7jVtF3Vb0pPyFfoURIodKzszRIgSJlo%3D&st=2017-12-15T02%3A40%3A59Z&se=2018-01-15T02%3A45%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/advisor?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C15%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712150253225962-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=3EDsPSVj%2FMhbjqc6o8vmDmqifWMZoDkHyIrItElxE2k%3D&st=2017-12-15T02%3A50%3A47Z&se=2018-01-15T02%3A55%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/network/application-gateway/probe?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C15%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712150319359743-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=k74FhVPdaZUrwY5112pUiGeno0k85vVN1lzlgtIiSCQ%3D&st=2017-12-15T03%3A16%3A58Z&se=2018-01-15T03%3A21%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[2017-03-09-profile/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/2017-03-09-profile/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/vm/image?view=azure-cli-2017-03-09-profile&branch=pr-en-us-565) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/2017-03-09-profile/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-2017-03-09-profile&branch=pr-en-us-565) (azure-cli-2017-03-09-profile) |
[latest/docs-ref-conceptual/TOC.yml](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/network/application-gateway/probe?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |
[latest/docs-ref-conceptual/azure-cli-configuration.md](https://github.com/sptramer/azure-docs-cli-python/blob/configuration/new/latest/docs-ref-conceptual/azure-cli-configuration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-565) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C15%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201712150325147036-565%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ARl%2FOK9jJ6TCF2STmhTBOHT61Fd9UHjzGwd6kCJa8qI%3D&st=2017-12-15T03%3A22%3A35Z&se=2018-01-15T03%3A27%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**